envolve-frontend/src/App.js

```diff
@@ -3,6 +3,7 @@ import './App.css';
 import {createMuiTheme, MuiThemeProvider} from "@material-ui/core";
 import UserContextProvider from "./context/UserContextProvider";
 import Navigation from "./components/Navigation";
+import {makeStyles} from "@material-ui/core/styles";
 
 
 const theme = createMuiTheme({
```

Comment:
You could define even more styles in your theme (like different typography variants) to reuse in your components :-)